themes/aframe/layout/partials/blog/twia_projects.ejs

```diff
@@ -0,0 +1,10 @@
+<% if (projects) { %>
+  <h3>Projects</h3>
+  <ul>
+    <% projects.forEach(function (project) { %>
+      <li>
+        <%- project.author %> built <a href="<%- project.url %>"><%- project.title %></a> - <%- project.description %>
```

Comment:
I was worried the number of links would detract from the actual projects. I think I can make these links but with less prominent styling.